scripts/ci/check_compliance.py

```diff
@@ -990,6 +1033,18 @@ def inheritors(klass):
     return subclasses
 
 
+def annotate(res):
+    """
+    https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#about-workflow-commands
+    """
+    if not isinstance(res, FmtdFailure):
+        return
```

Comment:
is this check needed?

Reply:
nope, will remove, it's a leftover, thanks.